_episodes/01-shell-basics.md

```diff
@@ -199,26 +196,25 @@ impact what external commands and libraries you are using.
 
 ### PATH - determines full path to the command to be executed
 
-Whenever running a command without providing its full path on the
+Whenever a command is run and its full path is not provided on the
 filesystem, the `PATH` environment variable is consulted to determine
-which paths to use to look for the command. You might have multiple
+which paths to use to look for the command. You may have multiple
 implementations or versions of the same command available at different
-locations, which might be specified within the PATH variable (separated
-with a colon). Although a very simple concept, it is a workhorse
-for "overlay distributions" (such as [conda](https://conda.io)), or "overlay
-environments (such as [virtualenv](https://virtualenv.pypa.io) in
-Python, or [modules](http://modules.sourceforge.net/) often used on
-HPC), and it is also
-the source of confusion in many cases where some unintended
-command is run instead. That is why any tool which aims to capture the
+locations, which may be specified within the PATH variable (separated
+with a colon). Although this is a very simple concept, it is a workhorse
+for "overlay distributions" (such as [conda](https://conda.io)). It is
+also a workhorse for "overlay environments” such as [virtualenv](https://virtualenv.pypa.io)
+in Python, or the often-used [modules](http://modules.sourceforge.net/) on
+HPC. Furthermore, it is also a source of much confusion in many cases where an unintended
+command is run instead. This is why any tool which aims to capture the
 state of the computational environment for later re-execution might
 need to store the value of the PATH variable to guarantee that even
 given the same set of files, the same commands are executed. For example,
 we may have two different versions of AFNI installed in different locations;
 without specifying the path to a particular installation of AFNI, we may
-unintentionally run a different version and end up with different results.
+unintentionally run a different version than intended and end up with different results.
 
-> ## How can you determine which command (full path to it) that you are about to use?
+> ## How can you determine which command (specifying a full path) you are about to use?
```

Comment:
well -- `specifying` is kinda out of place since it is a question "how to determine" not to specify